src/Illuminate/Support/Str.php

```diff
@@ -154,6 +154,10 @@ public static function camel($value)
      */
     public static function contains($haystack, $needles)
     {
+        if (! is_string($haystack)) {
```

Comment:
Breaking change. We should cast haystack to string in here. If the cast fails, we get an error, which is fine.

Reply:
Ok, will update it then. But from some other PRs I read before, I thought the typehint on the docblock would make this a non-BC.

Reply:
We allow stringish too.

Reply:
Fair enough, just pushed a new commit. Thanks